.env.sample

```diff
@@ -16,3 +16,7 @@ USER_SVC_DB_URI=
 JWT_SECRET=
 EMAIL_ADDRESS=
 EMAIL_PASSWORD=
+
+## Matching service variables
+MATCHING_SVC_PORT=6969
```

Comment:
Let's remove ports that are not hardcoded from the `.env.sample` file, technically `FRONTEND_PORT` value can also be removed.